test/resources/index.ts

```diff
@@ -1,14 +1,10 @@
+import { debug } from "debug";
 import { Fraction, TestCaseInput, Solution, TestCase } from "./models";
 import { BigNumber } from "ethers";
 import _ from "lodash";
 
-export const solveTestCase = function (
-  testCaseInput: TestCaseInput,
-  debug = false,
-): Solution {
-  // eslint-disable-next-line @typescript-eslint/no-empty-function, @typescript-eslint/explicit-function-return-type, @typescript-eslint/no-explicit-any
-  const log = debug ? (...a: any) => console.log(...a) : (): void => {};
-
+const log = debug("index.ts");
```

Comment:
`debug` uses `:` as separators. Also I think it makes more sense to use module names instead of files. Specifically there can be many `index.ts` since that is an "index" module for module resolution: `import "resources"` <=> `import "resources/index.ts"`.

```suggestion
const log = debug("test:resources");
```

And then submodules could just add `:submodulename`. So for example if there was a `test/resources/util.ts`, then its logger could use:

```ts
const log = debug("test:resources:util");
```